CleanWeather/Scenes/CityDetails/CityDetailsPresenter.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CityDetailsPresenter.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 19/11/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+protocol CityDetailsPresenter: SpinnerPresenter, AlertPresenter {
+    func displayCityDetails(from: CityWeather)
+    func displayCityDetailsList(from detailsList: CityWeatherDetails)
```

Comment:
I'd name this: 

func displayCityDetails(_ cityDetails: CityWeather)
func displayCityForecast(_ forecast: CityWeatherDetails)